src/main/java/org/folio/rest/impl/TenantReferenceAPI.java

```diff
@@ -30,7 +30,7 @@ public class TenantReferenceAPI extends TenantAPI {
 
   @Override
   public Future<Integer> loadData(TenantAttributes attributes, String tenantId, Map<String, String> headers, Context vertxContext) {
-    log.info("postTenant");
+    log.debug("loadData:: Trying to load reference data for tenant {}", tenantId);
```

Comment:
Lets use info level for this particular method, because its some startup operation for entire application and we need to see what going on in info level

Reply:
also please what values of isLoadSample, isLoadReference (true or false) provided
also lets log deleteTenantByOperationId method in info level

Reply:
1. fixed issue
2. logs are added to both isLoadSample and isLoadReference methods
3. deleteTenantByOperationId is changed to info level